core/src/test/scala-3/cats/derived/FunctorTests.scala

```diff
@@ -1,7 +1,7 @@
 package cats.derived
 
 import cats.Functor
-import cats.derived.all._
+import cats.derived.given
```

Comment:
Wait, are you telling me that this imports also extension methods 🤔

Reply:
I believe so!

Reply:
Argh no!!! It's because the tests are in the same package 😂

Reply:
When we're porting the tests, maybe we should put them in a different package to make sure we pick stuff like this up?

Reply:
I guess so